MAINTAINING.md

```diff
@@ -0,0 +1,76 @@
+# RIOT Maintainer Guidelines
+
+This list presents a series of guidelines for maintainers, including technical and non-technical. The list is not exhaustive, and represents a baseline on things that should be ensured for contributions.
+
+Notes:
+- Any of the items in this list might be skipped, if clearly and logically articulated reasons are presented.
+- The order of the steps is meant to maximize efficiency and minimize overhead, with the philosophy that failing in step n makes steps n+x obsolete. However, this efficiency can depend on the quality of the submission itself. If the PR is clearly not in a reviewable state, for example due to poor code cleanliness or overall design, it might be more efficient to give the contributor some broader comments for improvement before further review.
+- This is a working document and any changes, additions, or other discussions are encouraged, via PRs raised against the document.
+
+The list of RIOT maintainers, and the areas of RIOT they each maintain, can be found [here](https://github.com/RIOT-OS/RIOT/wiki/Maintainers).
+
+
+## Technical guidelines
+
+### 1. - Review the fundamentals
+
+1.1 Does the reasoning for this PR make sense? Are the requirements and design goals clearly thought out and clearly expressed? Is the problem that the PR intends to solve clearly stated?  
```

Comment:
These three questions look a little bit busy in the [rendered form](https://github.com/danpetry/RIOT/blob/0ade8b43908efdfa2ad3821762c085f7f179507e/MAINTAINING.md). Maybe split them up linewise?

```markdown
1.1 Does the reasoning for this PR make sense? \
    Are the requirements and design goals clearly thought out and clearly expressed? \
    Is the problem that the PR intends to solve clearly stated? 
```

(apply this for other multi-question points also if you agree)

Reply:
done